go/plugins/googleai/googleai.go

```diff
@@ -11,73 +11,84 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"strings"
 	"sync"
 
 	"github.com/firebase/genkit/go/ai"
 	"github.com/firebase/genkit/go/genkit"
-	"github.com/firebase/genkit/go/internal"
 	"github.com/firebase/genkit/go/plugins/internal/gemini"
-	"github.com/firebase/genkit/go/plugins/internal/uri"
-	"github.com/google/generative-ai-go/genai"
-	"google.golang.org/api/iterator"
-	"google.golang.org/api/option"
+
+	"google.golang.org/genai"
 )
 
 const (
 	provider    = "googleai"
 	labelPrefix = "Google AI"
+
+	// supported model names
+	gemini15Flash                = "gemini-1.5-flash"
+	gemini15Pro                  = "gemini-1.5-pro"
+	gemini15Flash8b              = "gemini-1.5-flash-8b"
+	gemini20Flash                = "gemini-2.0-flash"
+	gemini20FlashLite            = "gemini-2.0-flash-lite"
+	gemini20ProExp0205           = "gemini-2.0-pro-exp-02-05"
+	gemini20FlashThinkingExp0121 = "gemini-2.0-flash-thinking-exp-01-21"
```

Comment:
Created these constants in order to avoid passing the `labelPrefix` to the model definition function and to be able to populate `ModelInfo.Label` when defining `supportedModels`.